frontend/src/utils/templates.ts

```diff
@@ -59,13 +59,64 @@ const replacePlaceholders = (baseText: string, replacementConfig: { [key: string
   return newText;
 };
 
-export const confirmationTemplate = (replaceConfig: { [key: string]: string | string[] | undefined }) => {
+export const confirmationTemplateSubmission = (replaceConfig: { [key: string]: string | string[] | undefined }) => {
+  const PCNS_URL = 'https://pcns-prod-master.apps.silver.devops.gov.bc.ca';
+  const BC_BANNER = 'https://coms.api.gov.bc.ca/api/v1/object/446ee8ee-e302-4cb8-b44d-24a1f583edba';
+  const BC_FOOTER = 'https://coms.api.gov.bc.ca/api/v1/object/853de44a-e62f-41f5-81fd-6eff6cb66d52';
```

Comment:
Should be a single set of constants in this file instead of duplicated in each function